libs/ui/src/components/atoms/typography/typography.stories.tsx

```diff
@@ -47,9 +47,7 @@ export const AllFonts = () => {
   return (
```

Comment:
One thing I'm wondering about (and this may be a question we have with Design as well) is how we'd want to use this in a case where we want to have styles separate from semantic meaning. For example, if we want to have something that is semantically an `<h3/>` given where it is in the HTML layout tree, but we want the style to be that of what we have set here as `<H1/>` -- one case may be where we have a heading on a Card. In the page layout, it may be an `<h3/>` but given the hierarchy on the Card we'd want it to be sized as what we have as an `<H1/>` 

I've handled this before by using the `as` prop that's been supported in certain libraries where you could do something such as `<Heading as='h3' size='xxl'/>`

This is something I'm wondering and thinking about just generally with how we're thinking about these Typographic elements @jordanlesich @alexkeating @earth2travis

Reply:
Styled components supports the 'as' prop out of the box. So if we want a section text that is semantically different than the styling, we can edit it with that.